### PR TITLE
Update amazon-s3-connector-reference.adoc

### PR DESCRIPTION
**Pull request by patrik-patinak**

### CODE REVIEW

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 1]
```diff
@@ -29,20 +29,21 @@ endif::[]
...
 |===
 | Name | Type | Description | Default Value | Required
-| Connection Timeout a| Number |  |  +++50000+++ | 
-| Socket Timeout a| Number |  |  +++50000+++ | 
-| Access Key a| String |  |  | x
-| Secret Key a| String |  |  | x
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  +++false+++ | 
-| Max Connections a| Number |  |  | 
-| Region Endpoint a| String |  |  +++US_EAST_1+++ | 
+| S3 Compatible Storage URL a| String |  +++The URL to connect to in case an S3 compatible storage is used. If blank, the default AWS S3 one will be used.+++ |  |
```
Comment (dhirsch1) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: The URL to connect to when S3 compatible storage is used. If blank, the connector uses the default AWS S3 URL.

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 2]
```diff
@@ -29,20 +29,21 @@ endif::[]
...
-| Connection Timeout a| Number |  |  +++50000+++ | 
-| Socket Timeout a| Number |  |  +++50000+++ | 
-| Access Key a| String |  |  | x
-| Secret Key a| String |  |  | x
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  +++false+++ | 
-| Max Connections a| Number |  |  | 
-| Region Endpoint a| String |  |  +++US_EAST_1+++ | 
+| S3 Compatible Storage URL a| String |  +++The URL to connect to in case an S3 compatible storage is used. If blank, the default AWS S3 one will be used.+++ |  | 
+| Connection Timeout a| Number |  +++The amount of time to wait (in milliseconds) when initially establishing a connection before giving up and timing out. A value of 0 means infinity, and is not recommended.+++ |  +++50000+++ | 
+| Socket Timeout a| Number |  +++The amount of time to wait (in milliseconds) for data to be transferred over an established, open connection before the connection is timed out. A value of 0 means infinity,and is not recommended.+++ |  +++50000+++ |
```
Comment (dhirsch1) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: "...before the connection times out. A value of 0 means infinity and is not recommended."

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 3]
```diff
@@ -29,20 +29,21 @@ endif::[]
...
 | Name | Type | Description | Default Value | Required
-| Connection Timeout a| Number |  |  +++50000+++ | 
-| Socket Timeout a| Number |  |  +++50000+++ | 
-| Access Key a| String |  |  | x
-| Secret Key a| String |  |  | x
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  +++false+++ | 
-| Max Connections a| Number |  |  | 
-| Region Endpoint a| String |  |  +++US_EAST_1+++ | 
+| S3 Compatible Storage URL a| String |  +++The URL to connect to in case an S3 compatible storage is used. If blank, the default AWS S3 one will be used.+++ |  | 
+| Connection Timeout a| Number |  +++The amount of time to wait (in milliseconds) when initially establishing a connection before giving up and timing out. A value of 0 means infinity, and is not recommended.+++ |  +++50000+++ |
```
Comment (dhirsch1) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: "...before the connector gives up and times out." A value of 0 means infinity and is not recommended.

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 4]
```diff
@@ -29,20 +29,21 @@ endif::[]
...
-| Try Default AWSCredentials Provider Chain a| Boolean |  |  +++false+++ | 
-| Max Connections a| Number |  |  | 
-| Region Endpoint a| String |  |  +++US_EAST_1+++ | 
+| S3 Compatible Storage URL a| String |  +++The URL to connect to in case an S3 compatible storage is used. If blank, the default AWS S3 one will be used.+++ |  | 
+| Connection Timeout a| Number |  +++The amount of time to wait (in milliseconds) when initially establishing a connection before giving up and timing out. A value of 0 means infinity, and is not recommended.+++ |  +++50000+++ | 
+| Socket Timeout a| Number |  +++The amount of time to wait (in milliseconds) for data to be transferred over an established, open connection before the connection is timed out. A value of 0 means infinity,and is not recommended.+++ |  +++50000+++ | 
+| Access Key a| String |  +++The access key provided by Amazon.+++ |  | x
+| Secret Key a| String |  +++The secret key provided by Amazon.+++ |  | x
+| Try Default AWSCredentials Provider Chain a| Boolean |  +++Set this field to true to obtain credentials from AWS environment, See: https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html+++ |  +++false+++ |
```
Comment (dhirsch1) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: "...from the AWS environment..."

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc [hunk 5]
```diff
@@ -29,20 +29,21 @@ endif::[]
...
+| Max Connections a| Number |  +++Sets the maximum number of allowed open HTTP connections.+++ |  | 
+| Region Endpoint a| String |  +++Set the topic region endpoint.+++ |  +++US_EAST_1+++ | 
 | Reconnection a| <<Reconnection>> |  +++When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy+++ |  | 
-| Host a| String |  |  | 
-| Port a| Number |  |  | 
-| Username a| String |  |  | 
-| Password a| String |  |  | 
-| Domain a| String |  |  | 
-| Workstation a| String |  |  | 
+| Host a| String |  +++The optional proxy host+++ |  |
```
Comment (dhirsch1) [inline modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Please add periods after the field descriptios.

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 1 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Changed

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 2 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: I've chosen the second option ("before the connector gives up and times out.") in both cases

Comment (patrik-patinak) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Done for Socket Timeout

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 3 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Updated

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 4 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: done

File: modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc (hunk 5 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc]: Done

**Comment by george-hoffer** [comment]
Replaced by https://github.com/mulesoft/docs-connectors/pull/317

**Pull request closed without merging**